File: Day1/Answers/q5.py
# 5) Create a program to print the smallest and the largest number in the array/list [2,7,8,23,75,23,64].

# Easy answer---------------------------------------------------------
myList = [2,7,8,23,75,23,64]
myList.sort()
print(f"Largest = {myList[-1]}")
print(f"Smallest = {myList[0]}")


# Medium answer---------------------------------------------------------
myList = [2,7,8,23,75,23,64]

largest = myList[0]
smallest = myList[0]
for item in myList:
    if item > largest:
        largest = item
    if item < smallest:
        smallest = item
print(f"Largest = {largest}")
print(f"Smallest = {smallest}")

File: Day1/Answers/q5.c
// 5) Create a program to print the smallest and the largest number in the array/list [2,7,8,23,75,23,64].

# include <stdio.h>
int main(){
    int myArray[] = {2,7,8,23,75,23,64};
    int myArraylength = sizeof(myArray)/sizeof(int);
    int largest = myArray[0];
    int smallest = myArray[0];
    for (int i = 0; i < myArraylength; i++)
    {
        if(myArray[i]>largest){
            largest = myArray[i];
        }
        if(myArray[i]<smallest){
            smallest = myArray[i];
        }
    }
    printf("largest is %d\n",largest);
    printf("smallest is %d\n",smallest);
    return 0;
}